include/rtosc/miditable.h

```diff
@@ -60,13 +60,13 @@ class MidiMapperStorage
             T operator[](int i) const {assert(i>=0 && i<n);return t[i];}
 
             TinyVector insert(const T &t_)
-            {TinyVector next(n+1); for(int i=0;i<n; ++i) next.t[i]=t[i]; next.t[n] = t_;return std::move(next);}
+            {TinyVector next(n+1); for(int i=0;i<n; ++i) next.t[i]=t[i]; next.t[n] = t_;return next;}
```

Comment:
When testing this, I could not even confirm with the debugger that the move CTOR is called, but I guess this is due to RVO (return value optimization), even in debug mode.